### PR TITLE
Changed setframetype with setmode

### PR DESCRIPTION
**Pull request by VHolonec**

### CODE REVIEW

File: examples/first-frame/main.cpp [hunk 1]
```diff
@@ -243,7 +243,7 @@ int main(int argc, char *argv[]) {
         //TO DO: convert mode number/name once getFrameTypeDetails is added in the public api
     }
 
-    status = camera->setFrameType(modeName);
+    status = camera->setMode(modeName);
     if (status != Status::OK) {
         LOG(ERROR) << "Could not set camera frame type!";
```
Comment (dNechita) [inline examples/first-frame/main.cpp:248]: The error message hasn't been updated. It still refers to "frame type".

File: examples/data_collect/main.cpp [hunk 2]
```diff
@@ -416,7 +416,7 @@ int main(int argc, char *argv[]) {
     std::string sensorName;
     status = depthSensor->getName(sensorName);
 
-    status = camera->setFrameType(modeName);
+    status = camera->setMode(modeName);
     if (status != Status::OK) {
         LOG(ERROR) << "Could not set camera frame type!";
```
Comment (dNechita) [inline examples/data_collect/main.cpp:421]: "frame type" -> "mode"

File: bindings/python/aditofpython.cpp [hunk 3]
```diff
@@ -282,8 +280,7 @@ PYBIND11_MODULE(aditofpython, m) {
                 return status;
             },
             py::arg("availableModes"))
-        .def("setFrameType", &aditof::Camera::setFrameType,
-             py::arg("frameType"))
+        .def("setMode", &aditof::Camera::setMode, py::arg("frameType"))
```
Comment (dNechita) [inline bindings/python/aditofpython.cpp:283]: "frameType" -> "mode"

File: bindings/python/aditofpython.cpp [hunk 4]
```diff
@@ -602,7 +599,7 @@ PYBIND11_MODULE(aditofpython, m) {
         .def("getFrameTypeDetails",
              &aditof::DepthSensorInterface::getFrameTypeDetails,
              py::arg("frameName"), py::arg("details"))
-        .def("setFrameType", &aditof::DepthSensorInterface::setFrameType,
+        .def("setMode", &aditof::DepthSensorInterface::setMode,
              py::arg("details"))
```
Comment (dNechita) [inline bindings/python/aditofpython.cpp:603]: "details" -> "mode"

File: bindings/opencv/imshow/main.cpp [hunk 5]
```diff
@@ -103,7 +103,7 @@ int main(int argc, char *argv[]) {
         return 0;
     }
 
-    status = camera->setFrameType("lr-qnative");
+    status = camera->setMode("lr-qnative");
     if (status != Status::OK) {
         LOG(ERROR) << "Could not set camera frame type!";
```
Comment (dNechita) [inline bindings/opencv/imshow/main.cpp:108]: "frame type" -> "mode"

**Review by rbudai98 — APPROVED** [review]

**Review by dNechita — APPROVED** [review]

**Pull request merged**